app/views/shared/export/_plan.erb

```diff
@@ -32,55 +32,59 @@
+        <div style="page-break-before:always;"></div> <!-- Page break before each phase -->
+        <h1><%= download_plan_page_title(@plan, phase, @hash) %></h1>
+        <hr />
+        <% phase[:sections].each do |section| %>
+          <%# Only render if @show_sections_questions is true and either section not customised or section is customised and @show_custom_sections is true %>
+          <% if @show_sections_questions && (!section[:modifiable] || (@show_custom_sections && section[:modifiable]))  %>
+            <h3><%= section[:title] %></h3>
+          <% end %>
 
-        <% section[:questions].each do |question| %>
-          <div class="question">
-            <% if @show_sections_questions && !@public_plan %>
-              <p><%= raw question[:text].gsub(/<tr>(\s|<td>|<\/td>|&nbsp;)*(<\/tr>|<tr>)/,"") if question[:text].present?%></p>
-              <br>
-            <% end %>
-            <% answer = @plan.answer(question[:id], false) %>
-            <% blank = answer.present? ? answer.is_blank? : true %>
-            <% options = answer.present? ? answer.question_options : [] %>
-            <%# case where question has not been answered sufficiently to display%>
-            <% if @show_unanswered && (answer.blank? || (options.blank? && blank))%>
-              <p><%= _('Question not answered.') -%></p>
-            <% else %>
-              <%# case where Question has options %>
-              <% if options.present?%>
-                <ul>
-                  <% options.each do |opt| %>
-                    <li><%= opt.text %></li>
-                  <% end %>
-                </ul>
+          <% section[:questions].each do |question| %>
+            <div class="question">
+              <%# Only render if @show_sections_questions is true and either section not customised or section is customised and @show_custom_sections is true %>
+              <% if @show_sections_questions && (!section[:modifiable] || (@show_custom_sections && section[:modifiable])) && !@public_plan %>
+                <p><%= raw question[:text].gsub(/<tr>(\s|<td>|<\/td>|&nbsp;)*(<\/tr>|<tr>)/,"") if question[:text].present?%></p>
+                <br>
               <% end %>
-              <%# case for RDA answer display %>
-              <% if question[:format].rda_metadata? && !blank %>
-                <% ah = answer.answer_hash %>
-                <% if ah['standards'].present? %>
+              <% answer = @plan.answer(question[:id], false) %>
+              <% blank = answer.present? ? answer.is_blank? : true %>
+              <% options = answer.present? ? answer.question_options : [] %>
+              <%# case where question has not been answered sufficiently to display%>
+              <% if @show_unanswered && (answer.blank? || (options.blank? && blank))%>
+                <p><%= _('Question not answered.') -%></p>
+              <% else %>
+                <%# case where Question has options %>
+                <% if options.present?%>
```

Comment:
No need to check if `options.present?`. It should always be an Array, so we can remove this conditional .

Alternatively, if you don't want an empty `<ul>` tag, use `options.any?` to make it clearer that `options` is an Enumerable object.